docs/application/web/api/5.0/device_api/mobile/tizen/feedback.html

```diff
@@ -237,12 +240,11 @@ <h2>2. Interfaces</h2>
 <div class="brief">
  This interface defines what is instantiated by the <em>Tizen</em> object from the Tizen platform.
           </div>
-<pre class="webidl prettyprint">    [NoInterfaceObject] interface FeedbackManagerObject {
-        readonly attribute <a href="#FeedbackManager">FeedbackManager</a> feedback;
-    };</pre>
-<pre class="webidl prettyprint">    <a href="tizen.html#Tizen">Tizen</a> implements <a href="#FeedbackManagerObject">FeedbackManagerObject</a>;</pre>
-<p><span class="version">
-            Since: </span>
+<pre class="webidl prettyprint">  [NoInterfaceObject] interface FeedbackManagerObject {
+      readonly attribute <a href="#FeedbackManager">FeedbackManager</a> feedback;
+  };</pre>
+<pre class="webidl prettyprint">  <a href="tizen.html#Tizen">Tizen</a> implements <a href="#FeedbackManagerObject">FeedbackManagerObject</a>;</pre>
+<p><span class="version">Since: </span>
  3.0
           </p>
 <div class="description">
```

Comment:
The <em>tizen.feedback</em> object allows access to the Feedback API functionality.